tests/fail/both_borrows/newtype_pair_retagging.tree.stderr

```diff
@@ -7,7 +7,8 @@ LL |     unsafe { __rust_dealloc(ptr, layout.size(), layout.align()) }
    = help: this indicates a potential bug in the program: it performed an invalid operation, but the Tree Borrows rules it violated are still experimental
    = help: the accessed tag <TAG> is foreign to the protected tag <TAG> (i.e., it is not a child)
    = help: the access would cause the protected tag <TAG> to transition from Frozen to Disabled
-   = help: this is a loss of read permissions, which is not allowed for protected tags
+   = help: this transition would be a loss of read permissions, which is not allowed for protected tags
+   = help: the write access that caused this error was inserted implicitly due to a deallocation
```

Comment:
Correspondingly, here the message should say "deallocation through \<TAG> forbidden"